code/modules/mob/living/carbon/carbon.dm

```diff
@@ -262,6 +262,10 @@
 		unEquip(item)
 	if(src.client)
 		src.client.screen -= item
+	if(src.disabilities && CLUWNEMUT && prob(50))
```

Comment:
It appears only src can be properly used there, no other method unless you want to add a useless typecast